resources/carbon_cache.rb

```diff
@@ -21,14 +21,10 @@
 property :config, [Hash, nil], default: nil
 
 action :create do
-  python_package backend_name do
-    backend_attributes.each { |attr, value| send(attr, value) }
```

Comment:
This just some ruby/chef patterns that I don't really understand, please let me know what this is doing and how I can emulate this in my changes

Reply:
This is “just” collecting all the attributes and sending them though. It’s a little terse, and hard to read. So I’m with you here and write something easier to read.

Reply:
so its another way to set the properties [here](https://github.com/poise/poise-python#properties-3)? if `install_options` was set in the `backend_attributes` would it overwrite the `--no-binary=:all:` property below this? keeping that in mind for how I want to fix this